src/Miunie.Core/Providers/ITimeManipulationProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Miunie.Core.Providers
+{
+    public interface ITimeManipulationProvider
+    {
+        TimeSpan? GetTimeSpanFromString(string timeframe, int units);
+        (DateTime createdLocalTime, DateTime? editedLocalTime) GetMessageTimesLocalToUser(DateTime createdTime, DateTime? editedTime, MiunieUser user);
```

Comment:
Since this method has **two inputs** (created, edited) and **two outputs**, I think it's doing a bit two much.

I'd suggest having this method for a single DateTime? and just calling it twice.

Am I making sense? 😅

Reply:
Okay, I will adjust my code